remote-query/remote-query-server/src/main/java/org/infinispan/query/remote/impl/logging/Log.java

```diff
@@ -52,12 +49,12 @@
    @Message(value = "Failed to parse proto file : %s", id = 28011)
    CacheException failedToParseProtoFile(String fileName, @Cause Throwable cause);
 
-   @LogMessage(level = WARN)
-   @Message(value = "Using deprecated legacy indexing mode (all fields) for message type '%s' missing indexing" +
-         " annotations defined in file '%s'. Please annotate your message type for indexing or add" +
-         " 'option indexed_by_default = false;' to your schema file to disable indexing of message types" +
-         " that do not have indexing annotations.", id = 28012)
-   void legacyIndexingIsDeprecated(String typeName, String fileName);
+//   @LogMessage(level = WARN)
+//   @Message(value = "Using deprecated legacy indexing mode (all fields) for message type '%s' missing indexing" +
+//         " annotations defined in file '%s'. Please annotate your message type for indexing or add" +
+//         " 'option indexed_by_default = false;' to your schema file to disable indexing of message types" +
+//         " that do not have indexing annotations.", id = 28012)
+//   void legacyIndexingIsDeprecated(String typeName, String fileName);
```

Comment:
Remove?

Reply:
sure. but I think we had the rather strange rule of not removing unused log messages, just commenting them out. not sure why. I don't give a :rat:'s :ass: about them. will remove!

Reply:
Commenting them is better. That way we can't accidentally re-use an old ID.